### ISSUE TITLE
Recover the no of active cases by county

### ISSUE THREAD

**Issue by m6tlik** [koroonakaart/koroonakaart#236]
After update of the county statistics, the No of active cases (as sum of last 14 days) has been removed, though this is the best view of actual status. Overall cumulative trends finally show only the size of the county populations already long time.
The option could be recovered, as an average or total of last 14 days. Indexing can be also applied, but I am not 
sure that county population stats are sufficiently valid in real life.

Also note that "haigusjuhud" is incorrect use, as this is "nakatumisi" - Illness assume also symptoms, which is not always recorded.

**Comment by neuroactive** [comment]
Thanks for the feedback.

I removed the active cases estimate from the "Cases by county" chart for a few reasons. In general, I don't think it's a good idea to mix cumulative statistics with statistics which show a current state on the same chart. There are still some other charts in the app that do this, but I plan to separate those out into different charts when I find time.

I also have some doubts about how scientific the "Active cases estimate" is. The 14-day average infection time is very approximate and I'm not sure it takes into account those who are ill with Covid for a longer period. It also doesn't reflect the many people who are infected but never get diagnosed, either because they are asymptomatic or because they never seek medical treatment or get tested. So the true number of infected people is almost certainly considerably higher than the "active cases estimate", potentially many times higher. There is a reasonable argument therefore that publishing that statistic is misleading and potentially dangerous as it could lead people to believe that there are fewer infected people around than there actually are. As it stands, the "14-day cases per 100,000" figure is still visible from the map, which is essentially the same as "Active cases estimate per 100,000". That said, all the charts are getting reviewed at the moment. I agree with you that it's useful to show how the level of infection is changing in each county, but I will try to do so in a way that isn't potentially misleading.

I'm in the process of reviewing the terminology on the site, which will also involve updating the translations. I take your point about "Haigusjuhud" and will keep your feedback in mind as we do that.

**Comment by m6tlik** [comment]
You are right on absolute estimates, but present status estimates are of little value, when general range of the index or statistics is generally not known, undefined or the ranges change constantly depending on various reasons (new lines of corona ect). Actually, the cumulative/average number of detected infections per 14 days is mostly used and useful to see trends in the status. Map of index is very useful to see present comparison is space, but trends were only seen in this "potential actual no" or "average of 7/14 days graps) to see, where is quicker changes, where slower and where stable (e.g. it was useful to see that Tallinn dynamics in time is sharp and affects most of state level trends while in contrast Tartu has has been very slow and reaction too what even policy measures). Saying that, I have to note that many present cumulative graphics and one moment distributions are of little use in understanding dynamics for the end users and look just fun for ones who like to count. Another way, what is the purpose of use of those beyond presentation as such. Maybe this comment helps in the review of graphics.

**Comment by neuroactive** [comment]
Thanks for your comments. I broadly agree with the points you are making. I am planning to make some design improvements. One of the main aims will be to make the information which is presented more relevant and useful to the end user, as well as to improve the UX and give better structure to the information.

I am a little tied up at the moment dealing with various other issues, but as soon as I find time I will add a chart showing the 14-day infection rate per 100,000 broken down by county. Adding a chart showing the 7-day infection rate might also be helpful, as infection rates can change fast and so that should provide a more accurate indication of the infection level at a specific time.

I see that Terviseamet is now using the term "Nakkusjuht" on its own Covid dashboard, which seems like an improvement on "Haigusjuht", so we will probably switch to that.

**Comment by m6tlik** [comment]
Thanks, sounds great. 
I do use myself both 7 and 14 d running windows to be sure about short-term and general trends. 
Maybe 7 and 14 days windows can be used on the same graph with switch. 
But, both are quite useful indicators, absolute number and the scaled-one to 100k, as the first allows to pinpoint on the main counties of problem affecting the state level, the scaled one helps to compare trends and infection rate. 

Yes, "nakkusjuht" is novel short term for more common longer one.
